pkg/provider/chef/chef.go

```diff
@@ -0,0 +1,199 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package chef
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/url"
+	"regexp"
+
+	"github.com/go-chef/chef"
```

Comment:
```suggestion
	"github.com/chef/go-chef"
```
I suggest we switch to the fork of go-chef that is maintained by the chef team, it looks to have more recent and regular releases

Reply:
ok @welshDoug we will update it.